### PR TITLE
[0.12-branch] Validate if SM version supports OVNK as part of diagnose

### PR DESCRIPTION
**Pull request by sridhargaddam**
Submariner supports OVNKubernetes CNI starting from version
0.13.0 release. This PR adds basic check if the Submariner
version deployed in the cluster is the minimum supported version
or not.

Signed-off-by: Sridhar Gaddam <sgaddam@redhat.com>

### CODE REVIEW

**Comment by sridhargaddam** [comment]
@tpantelis @skitt @nyechiel This PR is not a backport but a simple validation added as part of `subctl diagnose cni` command. The motivation for adding this validation is that `subctl diagnose cni` is reporting OVNKubernetes as a supported CNI even when using Submariner 0.12 release and this is creating confusion to the users. I've seen two to three users running into similar situation during this week.

**Review by tpantelis — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -16,6 +16,7 @@ require (
 	github.com/hashicorp/go-version v1.3.0 // indirect
 	github.com/mailru/easyjson v0.7.7 // indirect
 	github.com/mattn/go-isatty v0.0.14
+	github.com/mcuadros/go-version v0.0.0-20190830083331-035f6764e8d2
```
Comment (skitt) [inline go.mod]: This is unmaintained; please use https://github.com/coreos/go-semver instead if possible.

**Review by skitt — CHANGES_REQUESTED** [review]

**Review by aswinsuryan — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (sridhargaddam) [reply go.mod]: Sure, will update it.

**Comment by nyechiel** [comment]
@skitt can you review again please?

**Review by skitt — APPROVED** [review]

**Pull request merged**